README.md

```diff
@@ -20,8 +20,11 @@ Check the rules to see what SonarLint can do for you:
 - [TypeScript rules](https://rules.sonarsource.com/typescript)
 - [Python rules](https://rules.sonarsource.com/python)
 - [PHP rules](https://rules.sonarsource.com/php)
+- [HTML rules](https://rules.sonarsource.com/html)
 
-You will benefit from the following code analyzers: [SonarJS](https://redirect.sonarsource.com/plugins/javascript.html), [SonarTS](https://redirect.sonarsource.com/plugins/typescript.html), [SonarPython](https://redirect.sonarsource.com/plugins/python.html) and [SonarPHP](https://redirect.sonarsource.com/plugins/php.html)
+You will benefit from the following code analyzers: [SonarJS](https://redirect.sonarsource.com/plugins/javascript.html), [SonarTS](https://redirect.sonarsource.com/plugins/typescript.html), [SonarPython](https://redirect.sonarsource.com/plugins/python.html), [SonarPHP](https://redirect.sonarsource.com/plugins/php.html) and [SonarHTML](https://redirect.sonarsource.com/plugins/html.html).
+
+The full list of available rules is visible in the "SonarLint Rules" view in the explorer, where you can activate and deactivate rules to match your project's conventions. SonarLint will also show a code action on each issue to deactivate the appropriate rule.
```

Comment:
I suggest some minor adjustments:
The full list of available rules is visible in the "SonarLint Rules" view in the explorer, where you can activate and deactivate rules to match your conventions. SonarLint also provides a code action on each issue to quickly deactivate the corresponding rule.